app/frontend/src/features/profile/profilePage/ProfilePage.tsx

```diff
@@ -1,21 +1,20 @@
-import { Card, List, ListItem, makeStyles } from "@material-ui/core";
-import { Link, Route, Switch, useHistory, useParams } from "react-router-dom";
+import { Card, CircularProgress, List, ListItem, makeStyles } from "@material-ui/core";
+import { TabContext, TabPanel } from "@material-ui/lab";
+import React, { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
 
+import Alert from "../../../components/Alert";
 import TabBar from "../../../components/TabBar";
+import { User } from "../../../pb/api_pb";
 import {
-  aboutRoute,
   changeEmailRoute,
   changePasswordRoute,
   editHostingPreferenceRoute,
   editProfileRoute,
-  favoritesRoute,
-  homeRoute,
-  photosRoute,
-  profileRoute,
-  referencesRoute,
 } from "../../../routes";
+import { service } from "../../../service";
+import { getCurrentUser } from "../../../service/user";
```

Comment:
It's better to access this through the `service` import above even though it's more characters to type, since we auto mock the `service` module in the current test setup (rather than having to mock individual files in `service`) to make things easier for writing tests.